packages/@okta/vuepress-theme-prose/assets/css/components/_footer.scss

```diff
@@ -1,29 +1,20 @@
 $redesign_white: #FFFFFF;
-$redesign_pink: #DD2864;
 $redesign_footer_border: #1F3B54;
```

Comment:
Can we open a ticket to cleanup all of the leftover $redesign_* stuff from the previous redesign? @albertchen-okta @shawyu-okta @toddbeckett-okta

Reply:
https://oktainc.atlassian.net/browse/OKTA-416268

Reply:
Any reason we're not doing this now? I'd expect prior redesigns and unused styles to be wiped away.